resources/views/store/manage_vouchers.blade.php

```diff
@@ -27,14 +27,18 @@
                         </ul>
                     </div>
                 </div>
-                <div class="warning">
-                    @foreach( $family->getNoticeReasons() as $notices )
-                    <p class="v-spaced">
-                        <i class="fa fa-exclamation-circle" aria-hidden="true"></i>
-                        Warning: {{ $notices['count'] }} {{ str_plural($notices['entity'], $notices['count']) }}
-                        is {{ $notices['reason'] }}
-                    </p>
-                    @endforeach
+                <div class="alert-message warning">
```

Comment:
I think you're always producing this box, even if there's no message.
Wrap it in an `@if` based on !empty $family->getNoticeReasons()

Reply:
Thank you - was fixed in edit page but not here 😞